docker/1.2-1/base/Dockerfile.cpu

```diff
@@ -0,0 +1,98 @@
+ARG UBUNTU_VERSION=18.04
+ARG CUDA_VERSION=10.2
+
+FROM nvidia/cuda:${CUDA_VERSION}-base-ubuntu${UBUNTU_VERSION}
+
+ARG PYTHON_VERSION=3.7
+ARG PYARROW_VERSION=0.16.0
+ARG MLIO_VERSION=0.6.0
+ARG XGBOOST_VERSION=1.2.0
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+# Force stdin, stdout and stderr to be totally unbuffered. Good for logging
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
```

Comment:
Will this have any impact on performance ? minor maybe

Reply:
Interesting point. Those environment variables were kept from previous versions, e.g., https://github.com/aws/sagemaker-xgboost-container/blob/master/docker/1.0-1/base/Dockerfile.cpu#L32, and I didn't think to remove them.